resources/validations/ui/src/context/cli/index.ts

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env -S ts-node --script-mode
+
+// @ts-ignore
+import * as SaxonJS from 'saxon-js';
+
+import { SaxonJsSchematronValidatorGateway } from '../shared/saxon-js-gateway';
+import { CommandLineController } from './cli-controller';
+import { readStringFile } from './file-gateway.humble';
+
+const controller = CommandLineController({
+  readStringFile,
+  validateSchematron: SaxonJsSchematronValidatorGateway({
+    sefUrl: `file:///Users/dan/src/10x/fedramp-automation/resources/validations/ui/public/ssp.sef.json`,
+    // The npm version of saxon-js is for node; currently, we load the
+    // browser version via a script tag in index.html.
+    SaxonJS: SaxonJS,
+    baselinesBaseUrl: `file:///Users/dan/src/10x/fedramp-automation/baselines/rev4/xml`,
+    registryBaseUrl: `file:///Users/dan/src/10x/fedramp-automation/resources/xml`,
+  }),
+});
+
+controller.parse(process.argv);
+console.log(controller.opts());
```

Comment:
NB: Question, when running the application as instructed is this what emits `{}` before the validations count?

```sh
$ time npm run cli  "validate" ../test/demo/FedRAMP-SSP-OSCAL-Template.xml 

> cli
> ts-node src/context/cli/index.ts "validate" "../test/demo/FedRAMP-SSP-OSCAL-Template.xml"

{}
Found 46 assertions

real    0m5.744s
user    0m7.012s
sys     0m0.365s
```